.github/workflows/pr_master_test.yml

```diff
@@ -19,26 +19,28 @@ defaults:
 
 env:
   PYTHONWARNINGS: ignore::UserWarning
+  PYTHON_CORE_PKGS: wheel
```

Comment:
Why do we need wheel?

Reply:
We used to `pip install` everything (wheel and other packages) in a single command.  That led to traces that had the following "weirdness" (see, e.g., https://github.com/jsiirola/pyomo/runs/1287711763?check_suite_focus=true):
```
Using legacy 'setup.py install' for dill, since package 'wheel' is not installed.
Using legacy 'setup.py install' for pathos, since package 'wheel' is not installed.
Using legacy 'setup.py install' for pyyaml, since package 'wheel' is not installed.
Using legacy 'setup.py install' for python-louvain, since package 'wheel' is not installed.
Using legacy 'setup.py install' for et-xmlfile, since package 'wheel' is not installed.
Using legacy 'setup.py install' for ppft, since package 'wheel' is not installed.
# ...
```
This ensures that wheel is installed first, then we go and install all the packages that we want in the environment.

Reply:
Makes sense. I approve.